rdwatch/core/utils/raster_tile.py

```diff
@@ -9,6 +9,7 @@
 
 def get_raster_tile(uri: str, z: int, x: int, y: int) -> bytes:
     # logger.warning(f'SITE URI: {uri}')
+    logger.warning(f'Image URI: {uri}')
```

Comment:
```suggestion
    # logger.warning(f'SITE URI: {uri}')
```

Or if we want to keep this log, I think we should remove the commented line.

Reply:
No reason to keep it, it was when I was debugging the issues with the downloading and some images not coming through with the removal of stuff form the S3.